TaskAdressProject/src/AdressPJ.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class AdressPJ {
+    public static void main(String[] args) {
+        String name, address;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("What is your name?");
+        name = sc.nextLine();
+        System.out.println("Where are you live?, " + name);
+        address = sc.nextLine();
+        System.out.print("OK, you are:\n " + name + " from " + address);
+
```

Comment:
Please remove extra line

Reply:
Done!

Reply:
This is the name of the project on PC and I mistakenly used it here :)
I already changed it

Reply:
This is the name of the project on PC and I mistakenly used it here :)
I already changed it